### PR TITLE
Address a number of cpplint errors in src/backend/codegen

### PR DESCRIPTION
**Pull request by hardikar**
@armenatzoglou @karthijrk Please have a look - some stuff in the framework, utils and ExecVariableLlst.

### CODE REVIEW

File: src/backend/codegen/ExecVariableList_codegen.cc [hunk 1]
```diff
@@ -559,7 +563,7 @@ bool ExecVariableListCodegen::GenerateExecVariableList(
   //   slot->PRIVATE_tts_isnull[attno] = true;
   // }
 
-  // TODO: Convert these to loops
+  // TODO(shardikar): Convert these to loops
```
Comment (kuien) [inline src/backend/codegen/ExecVariableList_codegen.cc]: I like this comment, lol

**Comment by kuien** [comment]
LGTM

**Comment by armenatzoglou** [comment]
In https://github.com/greenplum-db/gpdb/pull/799, @karthijrk mentioned two style issues that we should fix in ExecVariableList:
* file names should all be lower case, e.g., ExecVariableList_codegen.cc should be execvariablelist_codegen.cc, and
* variable names needs to be in lower case, e.g., ExecVariableList_func should be execvariablelist_func.

I am not sure that this is going to help us, but you guys can decide.

Other than that, LGTM.

**Comment by hardikar** [comment]
cpplint.py didn't give any warnings on the naming. I though we wanted to keep the naming consistent with their names in Postgres code. If make them lower case, shouldn't it be exec_variable_list_codegen.cc and exec_variable_list_func. I'm not too hung up on this naming scheme, but I'd like it to be consistent. What do you think @karthijrk and @armenatzoglou ?

**Comment by karthijrk** [comment]
In codegen, at least we followed the convention of having lower case and separated by '_'. So for consistent I would like to keep that way. BTW, exec_variable_list_codegen.cc and exec_variable_list_func looks good to me 👍

**Comment by karthijrk** [comment]
LGTM 👍 . Thanks a lot @hardikar  for addressing this.

**Comment by armenatzoglou** [comment]
+1. Thanks

**Comment by hardikar** [comment]
Thanks for reviewing! Pushed dce266adac5569927d9bf566141aaea2e9cf16f7.

**Pull request closed without merging**